### ISSUE TITLE
traceid 如何添加

### ISSUE THREAD

**Issue by ryanwx** [go-kratos/kratos#593]
看到日志中有打印 traceid ，那么 traceid  应该如何添加，添加到 header 里吗？

**Comment by UnderTreeTech** [comment]
框架默认是mock trace,你接入zipkin就可以了。会自动添加到rpc的metadata及http的response header中

**Comment by aimpsmile** [comment]
grpc add metadata to key "kratos-trace-id"
eg:
 kratos-trace-id : "5ae02fa6775f056b:53fa82d1775f056b:5ae02fa6775f056b:1"
convert struct php eg:
  ["X-B3-TraceId"]=>
  string(16) "5ae02fa6775f056b"
  ["X-B3-SpanId"]=>
  string(16) "53fa82d1775f056b"
  ["X-B3-ParentSpanId"]=>
  string(16) "5ae02fa6775f056b"
  ["X-B3-Sampled"]=>
  string(1) "1"

**Comment by pingguodeli573365** [comment]
db 如何添加到trace中？
